util/test/checkChplInstall

```diff
@@ -104,6 +104,15 @@ log_info "Temporary test job directory: ${TMP_TEST_DIR}"
 function cleanup_tmp_dir()
 {
     log_debug "Removing ${TMP_TEST_DIR}"
+    local i
+    # sometimes on an NFS file system, rm needs to wait for the TEST_JOB exe file to close
+    for i in 1 2 3 4 5
```

Comment:
5 is an arbitrary guess that seems reasonable.  
We will see how many tries it really takes when some Jenkins jobs run on the Crays after this PR merges. Thats the only way to know.